packages/ng/popup-employee/service/user-popover.store.ts

```diff
@@ -16,7 +16,7 @@ export class LuUserPopoverStore implements ILuUserPopoverStore {
 
 	public get(id: number): Observable<LuUserPopover> {
 		if (this.#cache.has(id)) {
-			return this.#cache.get(id)!;
+			return this.#cache.get(id);
```

Comment:
woah

Reply:
it's a great news, no more not null assertion when `get` is guarded inside a `has` 😄